config/cli.go

```diff
@@ -71,10 +71,10 @@ func ForkCLIFlags(envPrefix string) []cli.Flag {
 			EnvVars: opservice.PrefixEnvVar(envPrefix, "NETWORK"),
 		},
 		&cli.BoolFlag{
-			Name:    InteropEnabledInForkModeFlagName,
-			Value:   false,
-			Usage:   "Enable interop in fork mode",
-			EnvVars: opservice.PrefixEnvVar(envPrefix, "FORK_WITH_INTEROP"),
+			Name:    InteropEnabledFlagName,
+			Value:   true, // enabled by default
```

Comment:
I might be remembering incorrectly, but I thought we decided that for fork mode interop would not be enabled by default.

Reply:
hmm yea i'm actually not sure, @jakim929 do you remember?


with interop being a big focus now, may be fine to enable by default, idk